supervisor/supervisor.go

```diff
@@ -107,6 +107,7 @@ func eventLogger(s *Supervisor, path string, events chan Event, retainCount int)
 						if err := enc.Encode(le); err != nil {
 							logrus.WithField("error", err).Error("containerd: write event to journal")
 						}
+						count++
```

Comment:
Shouldn't this be in a else clause to account for the case where the `Encode()` call fails.